eng/common/scripts/stress-testing/stress-test-deployment-lib.ps1

```diff
@@ -215,9 +216,56 @@ function DeployStressPackage(
     if ($LASTEXITCODE) {exit $LASTEXITCODE}
 
     $dockerBuildConfigs = @()
-    
+
     $genValFile = Join-Path $pkg.Directory "generatedValues.yaml"
     $genVal = Get-Content $genValFile -Raw | ConvertFrom-Yaml -Ordered
+    $releaseName = $pkg.ReleaseName
+    if ($RerunFailedJobs) {
+        $pods = kubectl get pods -n $pkg.namespace -o json | ConvertFrom-Json
```

Comment:
Can we move this into its own helper function? That will make the outer function smaller and it will be easier to tell which variables are set/updated from this scope that are important for the remaining function code.